django/db/models/aggregates.py

```diff
@@ -14,9 +14,9 @@ class Aggregate(Func):
     contains_aggregate = True
     name = None
 
-    def resolve_expression(self, query=None, allow_joins=True, reuse=None, summarize=False):
+    def resolve_expression(self, query=None, allow_joins=True, reuse=None, summarize=False, for_save=False):
         assert len(self.source_expressions) == 1
-        c = super(Aggregate, self).resolve_expression(query, allow_joins, reuse, summarize)
+        c = super(Aggregate, self).resolve_expression(query, allow_joins, reuse, summarize, False)
```

Comment:
I noted this too and wasn't sure if it was an error or not. Can you 'save' an aggregate?

```
Model.objects.update(salary=Max('salary')) # would this work?
```